packages/vaadin-text-field/src/vaadin-text-area.js

```diff
@@ -89,8 +89,33 @@ class TextAreaElement extends ElementMixin(TextFieldMixin(ControlStateMixin(Them
           -webkit-overflow-scrolling: touch;
         }
 
-        [part='value'] {
+        .textarea-wrapper {
+          display: grid;
+          flex-grow: 1;
+          align-self: stretch;
+        }
+
+        .textarea-wrapper::after {
+          content: attr(data-replicated-value) ' ';
+          white-space: pre-wrap;
+          visibility: hidden;
+        }
+
+        [part='value'],
+        [part='input-field'] ::slotted(textarea) {
           resize: none;
+          overflow: hidden;
+        }
+
+        [part='value'],
+        [part='input-field'] ::slotted(textarea),
+        .textarea-wrapper::after {
+          grid-area: 1 / 1 / 2 / 2;
```

Comment:
**note:** If IE11 is in the support list, then this implementation (based on CSS Grids) likely needs to be tested in real IE11 because it doesn't really well support CSS Grids.

Reply:
Good point. We don't support IE11 though (not in the monorepo). 
It makes sense to check this if we want to backport the fix.

Reply:
Yeah, for backporting I have to use another way to fix the problem on IE.